Tests/DuplicatedSDKTest/DuplicatedSDKTest/DuplicatedSDKTestApp.swift

```diff
@@ -0,0 +1,50 @@
+import ObjectiveC.runtime
+import Sentry
+import SwiftUI
+
+@main
+struct DuplicatedSDKTestApp: App {
+    
+    init () {
+        SentrySDK.start { options in
+            options.dsn = "https://a@o.sentry.io/1"
+        }
+    }
+    
+    var body: some Scene {
+        WindowGroup {
+            ContentView()
+        }
+    }
+}
+
+struct ContentView: View {
+    var body: some View {
+        VStack {
+            Text("Test SentrySDK Integrations")
+            Text("\(checkIntegrations())")
+                .accessibilityIdentifier("TEST_RESULT")
+        }
+        .padding()
+    }
+    
+    func checkIntegrations() -> Bool {
```

Comment:
`l`: I think a bit more a descriptive method name as `areSDKIntegrationsInstalledInSameImageAsSentrySDK` or adding a comment what this does would improve the readability of the code.